packages/@o3r/configuration/README.md

```diff
@@ -12,9 +12,11 @@ This package is an [Otter Framework Module](https://github.com/AmadeusITGroup/ot
 [![Stable Version](https://img.shields.io/npm/v/@o3r/configuration?style=for-the-badge)](https://www.npmjs.com/package/@o3r/configuration)
 [![Bundle Size](https://img.shields.io/bundlephobia/min/@o3r/configuration?color=green&style=for-the-badge)](https://www.npmjs.com/package/@o3r/configuration)
 
-This module contains configuration-related features (CMS compatibility, configuration override, store and debugging).
+This module contains configuration-related features (CMS compatibility, configuration override, store, and debugging).
 It comes with an integrated ng builder to help you generate configurations supporting the Otter CMS integration.
```

Comment:
This could honestly be handled in another task but we should refrain as much as possible to mention "the CMS" as it may not be clear what we mean by that. Or we should have one reference point where we define what we mean by it and link it whenever we mention "the CMS"